src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -293,6 +303,16 @@ public void setValidateThrows(boolean value) {
         validateThrows = value;
     }
 
+    /**
+     * Setter to control whether to validateInAnonymousClasses.
+     *
+     * @param value user's value.
+     * @since 6.0
```

Comment:
Put 10.13.0, it will be next release number

Reply:
Done @romani